src/auro-input.js

```diff
@@ -286,15 +283,20 @@ export default class AuroInput extends BaseInput {
         aria-describedby="${this.uniqueId}"
         aria-invalid="${!this.isValid}"
       />
-
       <label for=${this.id} class="${classMap(labelClasses)}">${this.required ? this.label : `${this.label} (optional)`}</label>
       ${this.isValid
         ? html`
           <p class="inputElement-helpText" id="${this.uniqueId}">${this.helpText}</p>
         ` : html`
           <p class="inputElement-helpText error" id="${this.uniqueId}" role="alert" aria-live="assertive">${this.getErrorMessage()}</p>
         `}
-      ${this.type === 'credit-card' ? this.processCreditCard() : undefined}
+        ${this.type === 'credit-card' ? this.processCreditCard() : undefined}
+        ${this.type === 'credit-card' && this.icon ? html`<auro-icon class="creditCard-icon" category="payment" name="credit-card"></auro-icon>` : undefined}
+        ${this.inputIconName === 'cc-amex' ? html`<auro-icon class="creditCard-icon" category="payment" name="cc-amex"></auro-icon>` : undefined}
```

Comment:
Given this update, we don't need https://github.com/AlaskaAirlines/auro-input/blob/jbaker/dynamicallyLoadIcon/%2336/src/auro-input.js#L14-L19 any longer, do we?